.stickler.yml

```diff
@@ -0,0 +1,6 @@
+linters:
+  eslint:
+    config: './.eslintrc.json'
+files:
+    ignore:
+      - 'tailwind.js'
```

Comment:
Setup appropriate code linter. 

Let's use the config file here (https://github.com/microverseinc/linters-config/blob/master/react-redux/.stickler.yml) instead. I see `tailwind.js` is ignored in `.eslintignore` already, you don't need to change the stickler file.

Please let the `./.eslintrc.json` be same content with this: https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json

Reply:
Stickler still gives errors if I haven't ignored the files in stickler configurations. It doesn't see the eslintignore. Also what is the difference between the eslint configs?

Reply:
Also I don't see a requirement to stick to a specific config if I am not ignoring necessary files.

Reply:
> Stickler still gives errors if I haven't ignored the files in stickler configurations.

Really? That's weird. 
Is there a commit that shows this? I couldn't find any commit with error on this PR. 

> Also what is the difference between the eslint configs?

Please, see for yourself. I post the link to the two config.

Reply:
All my previous projects which used stickler gives linter errors if I don't ignore those files. 
> Is there a commit that shows this? I couldn't find any commit with error on this PR.
You don't have to.

> Please, see for yourself. I post the link to the two config.
I know exactly the difference I asked about the essence of changes not the actual changes. 

**Still can't see the screenshots for these changes**